### ISSUE TITLE
Learning Rate Matters: Vanilla LoRA May Suffice for LLM Fine-tuning

### ISSUE THREAD

**Issue by Hiroki11x** [Hiroki11x/Papers#511]
## Links
https://www.alphaxiv.org/overview/2602.04998

## Author
IBM Research

## TLDR

<img width="854" height="465" alt="Image" src="https://github.com/user-attachments/assets/55949523-1dae-4dcd-b710-dd5ec0ba543e" />

<img width="851" height="449" alt="Image" src="https://github.com/user-attachments/assets/d8e2228b-2a34-4225-8712-3baae2bb443b" />
